src/main/java/bio/terra/service/job/StairwayLoggingHooks.java

```diff
@@ -6,6 +6,7 @@
 import bio.terra.stairway.StairwayHook;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
+import org.slf4j.MDC;
```

Comment:
My understanding is that `MDC` uses a thread local variable to store the its map. Is only one stairway logging hook active per thread in practice?

Reply:
yep, exactly